topcoffea/modules/utils.py

```diff
@@ -162,3 +162,27 @@ def get_hist_from_pkl(path_to_pkl,allow_empty=True):
     if not allow_empty:
         h = {k:v for k,v in h.items() if v.values() != {}}
     return h
+
+# Check if the contents of two dictionaries of lists agree
+# Assumes structure d = {k1: [i1,i2,...], ...}
+def dict_comp(in_dict1,in_dict2,strict=False):
+
+    def all_d1_in_d2(d1,d2):
+        agree = True
+        for k1,v1 in d1.items():
+            if k1 not in d2:
+                agree = False
+                continue
```

Comment:
Probably won't affect performance, but you could put `break` here, or even `return False`.

Reply:
Thanks, added a `break`.